src/checkbox/skin.css

```diff
@@ -74,6 +69,11 @@ governing permissions and limitations under the License.
   }
 }
 
+.spectrum-Checkbox .spectrum-Checkbox-checkmark,
+.spectrum-Checkbox .spectrum-Checkbox-partialCheckmark {
+  color: var(--spectrum-checkbox-checkmark-color);
+}
```

Comment:
Would rather see these nested as:

```css
.spectrum-Checkbox {
  .spectrum-Checkbox-checkmark,
  .spectrum-Checkbox-partialCheckmark {
    color: var(--spectrum-checkbox-checkmark-color);
  }
}
```

Reply:
hmm...I tested it with my [pen](https://codepen.io/jianliao/pen/gNNZOw?editors=0110). It does not work and validate it with https://jigsaw.w3.org/css-validator/validator. It reported as an error.

Reply:
@jianliao it will work here since we're using `postcss-nested`. Your pen needs to include that post-css plugin for nested syntax to work.

Reply:
Thx. PR and pen updated.